pkg/diagnostics/diagnostic_bundle_test.go

```diff
@@ -489,6 +502,7 @@ func TestGenerateManagementClusterBundle(t *testing.T) {
 		}
 
 		f := diagnostics.NewFactory(opts)
-		_, _ = f.DiagnosticBundleManagementCluster(spec, kubeconfig)
+		_, _ = f.DiagnosticBundleManagementCluster(bs_spec, kubeconfig)
```

Comment:
do we need to validate the return for the method?

Reply:
Yeah, I will fix it shortly.